CONTRIBUTING.md

```diff
@@ -46,11 +46,11 @@ first read the [Basics docs](doc/basics.md).**
 
 ### Finding something to fix/improve
 
-All issues on Clippy are mentored, if you want help with a bug just ask
-@Manishearth, @flip1995, @phansch or @yaahc.
+All issues on Clippy are mentored, if you want help simply ask @Manishearth, @flip1995, @phansch
+or @yaahc directly by mentioning them in the issue or over on [Zulip].
```

Comment:
```suggestion
All issues on Clippy are mentored, if you want help simply ask @Manishearth, @flip1995, @phansch
or @llogiq directly by mentioning them in the issue or over on [Zulip].
```

`@yaahc` isn't active, since she's focusing on the error handling working group.

It would be great if you could add a sentence like "This list may be out of date. For the currently active mentors click ``[here](https://github.com/rust-lang/highfive/blob/master/highfive/configs/rust-lang/rust-clippy.json#L3)``"